babel.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(api) {
```

Comment:
What is babel? Why do we need it?

Reply:
https://babeljs.io/

transpiles code

Reply:
@dereckquock do we need that with native? There's no browser compatibility to worry about.

Reply:
https://docs.expo.io/versions/v26.0.0/guides/using-modern-javascript/#polyfills-and-babel-transforms

most of the stuff is included 👍 
but there's other stuff like `optional-chaining`

Reply:
meh ok. I don't have many feelings either way.

Reply:
it doesn't add any bloat or anything, it just makes it so that we can safely use new JS stuffs